modules/tensor_mechanics/test/tests/2D_geometries/tests

```diff
@@ -1,25 +1,33 @@
 [Tests]
+  issues = '#5142'
   [./plane_strain]
     type = Exodiff
     input = 'planestrain.i'
     exodiff = 'planestrain_out.e'
     abs_zero = 1e-7
     compiler = 'CLANG GCC'
+    design = 'source/materials/ComputePlaneSmallStrain.md syntax/Modules/TensorMechanics/Master/index.md'
+    requirement = 'The ComputePlaneSmallStrain class shall, as created by the TensorMechanics MasterAction, compute the elastic stress and strain for a planar square geometry under tension using a total small plane strain formulation.'
```

Comment:
Is "as created by the TensorMechanics Master Action" really important? It seems like that is not really the core purpose of the test (because we're not running it both with and without that action or anything like that). I'd suggest removing that phrase from all of these tests.

Reply:
I'm not sure if it is important or not, but ComputePlaneSmallStrain (ect. ect) does not appear directly in the input file because I'm generating it with the tensor mechanics master action. Should I make a note in the input file that the master action generates the strain class?

Reply:
I think it's OK if we refer to a class that is created by an Action in the requirement.  As long as that class gets used in the analysis, we are testing that requirement.